system-addon/content-src/components/TopSites/TopSites.jsx

```diff
@@ -23,10 +23,19 @@ class TopSite extends React.Component {
       action_position: this.props.index
     }));
   }
+  _selectTileTitle(link) {
+    const title = link.pinTitle || shortURL(link);
```

Comment:
can you put this functionality inside the ```shortURL``` function since we want to keep these components as simple as possible